ui/webui/test/helpers/installer.py

```diff
@@ -26,19 +26,41 @@ class Installer():
     def __init__(self, browser):
         self.browser = browser
 
-    def begin_installation(self):
+    def begin_installation(self, should_fail=False, confirm_erase=True):
```

Comment:
Not in this case, but as you said there could be other reasons for the installation to not start. And I would like should_fail to become common parameter for functions for interacting with the webui. It will make development of future tests easier.

Reply:
That makes sense, thank you.